wpilibc/athena/src/PIDController.cpp

```diff
@@ -416,7 +422,7 @@ float PIDController::GetAvgError() const {
   {
     std::lock_guard<priority_recursive_mutex> sync(m_mutex);
     // Don't divide by zero.
-    if (m_buf.size()) avgError = m_setpoint - m_bufTotal / m_buf.size();
```

Comment:
It was probably this line that broke it, because this is computing the error relative to now, not relative to when the sample was taken.